actions/form/class.Instance.php

```diff
@@ -112,10 +112,13 @@ protected function initElements()
                 continue;
             }
 
-            //map properties widgets to form elments
-            $element = $this->getElementFactory()
-                ->withInstance($instance)
-                ->create($property);
+            //map properties widgets to form elements
+            $elementFactory = $this->getElementFactory();
+            if($instance != null) {
+                $elementFactory->withInstance($instance);
+            }
```

Comment:
PSR and strict check :) 

```suggestion
            $elementFactory = $this->getElementFactory();
            
            if ($instance instanceof core_kernel_classes_Resource) {
                $elementFactory->withInstance($instance);
            }
```

Reply:
Realized that after pushing :sweat: Fixed in a4d454316c584f602b3b5e65f5e4200112a59a7f

As you suggested, I'm  changing the `null` comparison with `instanceof` as well in d410e8f624d05ff2740c191857a4c033af6e2783